src/ProcessSlave.php

```diff
@@ -84,7 +84,7 @@ class ProcessSlave
     protected $inShutdown = false;
 
     /**
-     * @var BufferingLogger
+     * @var BufferingLogger|\Symfony\Component\Debug\BufferingLogger
```

Comment:
Really? Why not `Psr\Log\LoggerInterface`?

Reply:
I do not understand the business logic stuff just the syntax :)

Reply:
I've matched it to https://github.com/php-pm/php-pm/blob/8a1db9c8eeb6b605691cb3738fa31e2add78a882/src/Debug/BufferingLogger.php#L41

Reply:
They're both implementing the LoggerInterface- that should work better. The curse of automatic tools ;)

Reply:
Yikes- you're right since we're apparently using the non-interface `cleanLogs` method.